src/app.jsx

```diff
@@ -317,40 +417,62 @@ class Application extends React.Component {
         if (this.state.containers !== null) {
             Object.keys(this.state.containers).forEach(c => {
                 const container = this.state.containers[c];
-                const image = container.imageid;
+                const image = container.imageid + container.isSystem.toString();
                 if (imageContainerList[image]) {
                     imageContainerList[image].push({
                         container: container,
-                        stats: this.state.containersStats[container.id],
+                        stats: this.state.containersStats[container.id + container.isSystem.toString()],
                     });
                 } else {
                     imageContainerList[image] = [ {
                         container: container,
-                        stats: this.state.containersStats[container.id]
+                        stats: this.state.containersStats[container.id + container.isSystem.toString()]
                     } ];
                 }
             });
         } else
             imageContainerList = null;
 
-        let imageList;
-        let containerList;
-        imageList =
+        let startService = "";
+        if (!this.state.systemServiceAvailable && permission.allowed) {
+            startService = <div className="alert alert-info dialog-info">
+                <div className="info-message">
+                    <span className="fa fa-exclamation-triangle" />
+                    <span>{_("System Podman service is also available")}</span>
+                </div>
+                <button onClick={this.startService}>{_("Start")}</button>
+            </div>;
+        }
+        if (!this.state.userServiceAvailable && this.state.userServiceExists) {
+            startService = <div className="alert alert-info dialog-info">
+                <div className="info-message">
+                    <span className="fa fa-exclamation-triangle" />
+                    <span>{_("User Podman service is also available")}</span>
```

Comment:
Related to this: When I click the "start" button, I briefly see the empty state pattern with the check box that selects whether to permanently enable the service or not. But this goes away after a blink of an eye, back to the overview page. If this is meant to completely replace the empty state pattern, then the latter should not be triggered any more?

Reply:
It is not meant to replace the empty state pattern. There are 3 cases:
1. Neither service is running (show the empty pattern)
2. Only one service is running and other is present (load the one and show message about the other one)
3. Only one service is present and one is running (just load it normally and don't show anything)

> When I click the "start" button, I briefly see the empty state pattern with the check box that selects whether to permanently enable the service or not.

Hmm, I am not able to reproduce this, but I'll try a bit harder